src/CaptureFileInfo/CMakeLists.txt

```diff
@@ -48,8 +48,11 @@ target_link_libraries(
           QtUtils
           Qt5::Test)
 
-add_custom_command(TARGET CaptureFileInfoTests POST_BUILD
-  COMMAND ${CMAKE_COMMAND} -E copy_directory ${CMAKE_CURRENT_LIST_DIR}/testdata
-  $<TARGET_FILE_DIR:CaptureFileInfoTests>/testdata)
+add_custom_command(TARGET CaptureFileInfoTests POST_BUILD COMMAND ${CMAKE_COMMAND}
+          -E remove_directory $<TARGET_FILE_DIR:CaptureFileInfoTests>/testdata/CaptureFileInfo)
+
+add_custom_command(TARGET CaptureFileInfoTests POST_BUILD COMMAND ${CMAKE_COMMAND}
+          -E copy_directory ${CMAKE_CURRENT_SOURCE_DIR}/testdata/
+          $<TARGET_FILE_DIR:CaptureFileInfoTests>/testdata/CaptureFileInfo)
```

Comment:
When using two rules you don't have control over the execution order. They could be executed in parallel.

But you can use multiple COMMAND directives in the same `add_custom_command` call. They will be executed in order.

Reply:
I copied this from the CMakeLists.txt of OrbitBase. I fixed it here and added a separate commit for fixing it in OrbitBase

Reply:
Even better. Thanks!